include/RAJA/policy/hip/MemUtils_HIP.hpp

```diff
@@ -115,17 +115,18 @@ struct DeviceZeroedAllocator
   void* malloc(size_t nbytes)
   {
     auto res = ::camp::resources::Hip::get_default();
+    constexpr int zero = 0;
     void* ptr;
-    hipErrchk(hipMalloc(&ptr, nbytes));
-    hipErrchk(hipMemsetAsync(ptr, 0, nbytes, res.get_stream()));
-    hipErrchk(hipStreamSynchronize(res.get_stream()));
+    hipErrchk(hipMalloc, &ptr, nbytes);
+    hipErrchk(hipMemsetAsync, ptr, zero, nbytes, res.get_stream());
```

Comment:
Is it worth trying to name all the arguments? It seems likely that sooner or later someone will put the constant back in, either in this call or in a different call.

Reply:
I decided to give up and handle temporaries/literals in the arguments. I plan to remove the named argument changes.